source/_docs/clamav.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Enabling ClamAV for WordPress or Drupal
+description: Learn how to protect your site against malicious files with malware and other potential threats using ClamAV, an open source antivirus engine.
+tags: [clamav, security, files]
+categories: [security]
+---
+
+## What is ClamAV?
+
+**ClamAV** is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats. It supports multiple file formats, file and archive unpacking, and multiple signature languages.
```

Comment:
suggest editing this paragraph down to a brief 1-sentence overview, something like:

**ClamAV** is an open source antivirus engine your WordPress or Drupal site can use to scan user-uploaded files.